core/trino-main/src/test/java/io/trino/sql/planner/iterative/rule/TestSwapAdjacentWindowsBySpecifications.java

```diff
@@ -61,7 +61,7 @@ public void doesNotFireOnPlanWithSingleWindowNode()
                                 ImmutableList.of(p.symbol("a")),
                                 Optional.empty()),
                         ImmutableMap.of(p.symbol("avg_1"),
-                                new WindowNode.Function(resolvedFunction, ImmutableList.of(), DEFAULT_FRAME, false)),
+                                new WindowNode.Function(resolvedFunction, ImmutableList.of(), Optional.empty(), DEFAULT_FRAME, false)),
```

Comment:
We should adjust `SymbolsExtractor.extractAll(WindowNode.Function function)` to account for the ordering scheme. It is used on this code path.

I haven't checked all references to `WindowNode.Function`.

Reply:
went through those now. Seems ok